src/bin_utils.rs

```diff
@@ -0,0 +1,86 @@
+use std::{io, iter, num::ParseIntError};
```

Comment:
Whilst we're at it, why not add a doc comment explaining the module?

Reply:
Done!